content/es/docs/concepts/architecture/cloud-controller.md

```diff
@@ -0,0 +1,234 @@
+
+### 2. Kubelet
+
+El controlador de nodos incluye la funcionalidad del kubelet que es dependiente de la nube. Previa a la introducción de CCM, el kubelet era responsable de inicializar un nodo con detalles específicos al proveedor como direcciones IP, etiquetas de región/zona y tipo de instancia. La introduccion de CCM transfiere esta inicialización del kubelet al CCM.
+
+En este nuevo modelo, el kubelet inicializa un nodo sin información especifica del proveedor de servicios. Sin embargo, añade un `taint` al nodo recién creado de forma que este no esté disponible para el planificador hasta que el CCM completa el nodo con la información específica del proveedor. Sólo entonces elimina el `taint` y el nodo se vuelve accesible.
+
+## Mecanismo de Plugins
+
+El Cloud Controller Manager utiliza interfaces Go(lang) para permitir que implementaciones de los proveedores de servicios sean conectadas. Específicamente, utiliza el CloudProvider Interface definido [aquí](https://github.com/kubernetes/cloud-provider/blob/9b77dc1c384685cb732b3025ed5689dd597a5971/cloud.go#L42-L62).
+
+La implementación de los cuatro controladores referenciados en este documento, algunas estructuras de inicialización junto con el interface CloudProvider, permanecerán como parte del núcleo de Kubernetes. 
+
+## Autorización
+
+Esta sección divide el nivel de acceso requerido por varios objetos API para que el CCM pueda llevar acabo sus operaciones.
+
+### Controlador de Nodos
+
+El controlador de nodos sólo opera con objetos nodo. Necesita de acceso total para obtener, listar, crear, actualizar, arreglar, monitorizar y borrar objetos nodo.
+
+v1/Node:
+
+- Get
+- List
+- Create
+- Update
+- Patch
+- Watch
+- Delete
+
+### Controlador de Rutas
+
+El controlador de rutas escucha por eventos de creación de nodos y configura sus rutas. Necesita acceso a los objetos Node. 
+
+v1/Node:
+
+- Get
+
+### Controlador de Servicios
+
+El controlador de servicios escucha por eventos de creación, actualización y borrado de objetos servicio, y se encarga de configurar los endpoints para dichos servicios.
+
+Para acceder a los objetos servicio, necesita permisos para listar y monitorizar. Para el mantenimiento de servicios necesita permisos para arreglar y actualizar.
+
+Para configurar endpoints para los servicios necesita permisos para crear, listar, obtener, monitorizar y actualizar.
+
+v1/Service:
+
+- List
+- Get
+- Watch
+- Patch
+- Update
+
+### Otros
+
+La implementación del núcleo de CCM requiere acceso para crear eventos, y para asegurar la seguridad de operaciones; necesita acceso para crear ServiceAccounts.
+
+v1/Event:
+
+- Create
+- Patch
+- Update
+
+v1/ServiceAccount:
+
+- Create
+
+El RBAC ClusterRole para CCM se muestra a continuación:
```

Comment:
> se muestra a continuación:

por

> tiene la siguiente pinta:

Reply:
not a chance

Reply:
En inglés es exactamente `looks like this`

> The RBAC ClusterRole for the CCM **looks like this:**`

Pero te lo compro. xD